keyboards/dz60/keymaps/iso_vim_arrow/keymap.c

```diff
@@ -3,33 +3,36 @@
 #define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-/* ISO VIM arrow
-* ,-----------------------------------------------------------------------------------------.
-* | Esc |  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  ß  |  ´  |   BSPC    |
-* |-----------------------------------------------------------------------------------------+
-* | Tab    |  Q  |  W  |  E  |  R  |  T  |  Z  |  U  |  I  |  O  |  P  |  P  |  Ü  | *  |   |
-* |-------------------------------------------------------------------------------------|   +
-* | Layer_1    |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  Ö  |  Ä |  # |Enter |
-* |-----------------------------------------------------------------------------------------+
-* | Shift|  < |  Y  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  .  |  -  |  RShift   |
-* |-----------------------------------------------------------------------------------------+
-* | LCtrl |  LGUI  |  LAlt  |          Space           |  RAlt | Left |  Down | Up  | Right |
-* `-----------------------------------------------------------------------------------------'
-*/
-	LAYOUT(
-		KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_NO, KC_BSPC,
-		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
-		MO(1), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
-		KC_LSFT, KC_NUBS, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_NO,
-    KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, KC_LEFT, KC_DOWN, KC_UP, KC_RIGHT),
 
+  /* ISO VIM arrow (ISO German layout shown)
```

Comment:
I like the renaming, because it is more generic and reusable. File and this description still mention the word "vim arrow" which is a more specific example. This is intended, because it's an example or should it be changed, too?

Reply:
Perhaps I should change `ISO VIM arrow` on line 7 to `ISO 5x1u`.

I'm a bit unsure of which way I prefer. The code block you've pointed out shows the physical layout of the keymap, whose name is unchanged. I question whether `vim arrow` or `5x1u` is better, because to my mind, neither has name recognition. It's not like saying "Tsangan" or "Winkeyless", terms that are widespread in the community.

Something that I think is less than ideal is that the only way to view the physical layouts of the matrices are to view them via the [Configurator](https://config.qmk.fm/) &mdash; you can't view them directly in the repo unless someone documents them, and currently there's no standard for doing so.

Reply:
I would add more detail to the description and things would be clearer. E.g. changing the name to 5x1u, but the description and layout is describing a variant which is supposed to have arrow keys in a single line and can be used to navigate similar to the vim editor.

Reply:
I like that idea. I think I'll go with that.